setup.py

```diff
@@ -41,6 +41,9 @@ def get_description():
         # ]
         "ie": [
             "zipfile-deflate64"
+        ],
+        "si": [
+            "rarfile>=4.0"
```

Comment:
This and the si.py are probably unintentionally in here?

Reply:
Pfiew. I messed up somewhere with rebasing 🙈